paddle/operators/activation_op.cc

```diff
@@ -338,6 +338,38 @@ class ThresholdedReluOpMaker : public framework::OpProtoAndCheckerMaker {
   }
 };
 
+template <typename AttrType>
+class HardSigmoidOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  HardSigmoidOpMaker(framework::OpProto *proto,
+                     framework::OpAttrChecker *op_checker)
+      : OpProtoAndCheckerMaker(proto, op_checker) {
+    AddInput("X", "Input of HardSigmoid operator");
+    AddOutput("Y", "Output of HardSigmoid operator");
```

Comment:
Out [name convention documentation](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/name_convention.md) suggest us use the "(type, default value) usage" style to write the comments. I guess this comment can be further refined. And fc_op is a good example: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/fc_op.cc#L122.

Reply:
@lcy-seso  Do you recommend that I modify this for all ops in the activation.cc? Unfortunately all activations in this file do not follow the naming convention. The other option is that I merge this PR and open an issue to fix the naming convention of all Ops in Activations? The second option will keep this PR clean. What do you suggest?

Reply:
I prefer we guarantee all the newly written unmerged PR follows this naming convention first, and make a new PR later to fix all the old codes.

Reply:
Thank you for pointing this out. Opened a separate issue to fix this. https://github.com/PaddlePaddle/Paddle/issues/4772